apps/aevm/src/aevm_ae_primops.erl

```diff
@@ -536,12 +541,19 @@ crypto_call(Gas, Op, _Value, Data, State) ->
     case aevm_eeevm_state:vm_version(State) of
         ?VM_AEVM_SOPHIA_1 ->
             {error, out_of_gas};
-        VMVersion when ?IS_AEVM_SOPHIA(VMVersion), VMVersion >= ?VM_AEVM_SOPHIA_2 ->
+        VMVersion when ?IS_AEVM_SOPHIA(VMVersion) andalso
+                       VMVersion >= ?VM_AEVM_SOPHIA_2 andalso
+                       Op < ?PRIM_CALL_CRYPTO_ECVERIFY_SECP256K1 ->
```

Comment:
This is not a semantics change because:
* Code path reaching this already checked PRIM_CALL_IN_CRYPTO_RANGE
* All other opcodes included are smaller than this one.

Ok.